docs/creating-service-bindings/unbind-application.md

```diff
@@ -2,4 +2,10 @@
 sidebar_position: 5
 ---
 
-# Unbind Application
\ No newline at end of file
+# Unbind Application
+
+To unbind an application from a service, delete the `ServiceBinding` custom resource linked to it:
+
+```
+kubectl delete ServiceBinding binding-request
```

Comment:
This should be a bit more generic.
You can give a sample as you did, but that also need a generic command description.
```kubectl delete ServiceBinding <service-binding.metadata.name>```
where `service-binding.metadata.name` is the name used in the metadata describing your ServiceBinding resource.

Reply:
Yes. Makes sense.

Reply:
Addressed this 843de93726e8701e3c532113e6d31e87706f1c5c